.github/workflows/test-integration.yml

```diff
@@ -0,0 +1,42 @@
+name: Integration tests
+on:
+  push:
+    branches:
+      - devel
+      - alpha
+      - stable
+  pull_request:
+    branches:
+      - devel
+      - alpha
+      - stable
+    types:
+      - closed
+  schedule:
+    - cron: '20 16 * * *' # daily at 16:20 UTC
+jobs:
+  tests:
+    if: github.event.schedule || github.event.pull_request.merged
+
+    strategy:
+      matrix:
+        os: [ ubuntu-20.04, macos-11 ] # list of os: https://github.com/actions/virtual-environments
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v3
+      - run: git submodule update --init --recursive --force
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18.x
+      - name: Install dependencies
+        run: |
+          if [ "$RUNNER_OS" == "Linux" ]; then
+            sudo apt update && sudo apt install build-essential
+          fi
+        shell: bash
+      - uses: actions/cache@v3
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ matrix.os }}-go-${{ hashFiles('**/go.sum') }}
+      - run: make test-integration
```

Comment:
let's run on Win also - it more strict for catching - un-closed file descriptors leaks

Reply:
example: https://github.com/ledgerwatch/erigon/blob/71b819bad1ee60a6f7050591c6d8a08f21bcfe2a/.github/workflows/ci.yml#L55